### PR TITLE
TZERO: new analysis task

### PR DESCRIPTION
**Pull request by afurs**

### CODE REVIEW

File: PWGPP/CMakeLists.txt [hunk 1]
```diff
@@ -332,6 +334,10 @@ install(FILES HMPID/AddTaskHmpidQA.C DESTINATION PWGPP/HMPID)
 install(FILES HMPID/macros/copyHMPQAfiles.C
  	      HMPID/macros/makeHMPQA.C DESTINATION PWGPP/HMPID/macros)
 install(FILES T0/AddTaskT0QA.C
+        T0/spectrum/AddAnalysisTaskSpectrumT0.C
+        T0/spectrum/runListRCT2016_pp_tzero.root
```
Comment (chiarazampolli) [inline PWGPP/CMakeLists.txt:338]: These are small files, I guess it is fine to have them in AliPhysics. @shahor02, what do you think?

File: PWGPP/T0/spectrum/AliAnalysisTaskSpectrumT0.cxx [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+}
+/*******************************************************************************************************************/
+void AliAnalysisTaskSpectrumT0::UserCreateOutputObjects()	{
+  /// Create histograms
+  /// Called once
+  mOutputList = new TList();
+  mOutputList->SetOwner(kTRUE); /// Will delete the histos on cleanup
+  mOutputList->SetName("output");
+
+  #if !(defined(__CINT__) || defined(__MAKECINT__))
```
Comment (chiarazampolli) [inline PWGPP/T0/spectrum/AliAnalysisTaskSpectrumT0.cxx:81]: Why do you need this?

Comment (afurs) [reply PWGPP/T0/spectrum/AliAnalysisTaskSpectrumT0.cxx:81]: There are some problems during ROOT5 compilation( mostly with std library). So I decided to hide some parts of the code. I'm planning to use this task only with ROOT6

**Comment by chiarazampolli** [comment]
Hello @afurs ,

This is a T0-specific PR, maybe the best is that @AllaMaevskaya  reviews it? 
I have a question, though: where is this task meant to run? If in the QA, can you comment on the output size?

Chiara

**Comment by afurs** [comment]
Hello @chiarazampolli ,
I'm planning to run it within LEGO task. Output size is parametrized by some options(Number of runs and set of defined event classes).

**Comment by chiarazampolli** [comment]
Thanks @afurs . 
If the code is blessed by @AllaMaevskaya , i can approve it.

Chiara

**Review by AllaMaevskaya — APPROVED** [review]

**Comment by AllaMaevskaya** [comment]
For me OK

**Comment by afurs** [comment]
The problem is on Fluka side, and not related to PR's code.

**Comment by chiarazampolli** [comment]
+1

**Comment by chiarazampolli** [comment]
I approved, even if there is some issues with the CI (present in a all PRs in AliPhysics). I already pinged the experts in another one.

**Comment by afurs** [comment]
@chiarazampolli Thanks!

**Comment by chiarazampolli** [comment]
Hi @TimoWilken , @ktf ,

This PR did not re-run the CI, if I am not wrong... The error is still with Fluka. 

Chiara

**Pull request merged**